index.html

```diff
@@ -5,46 +5,20 @@
       <meta http-equiv="X-UA-Compatible" content="IE=edge">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <title>Learning</title>
-      <link rel="stylesheet" href="./public/css/Index.css">
-      <link rel="stylesheet" href="./public/css/task_one.css">
+      <link rel="stylesheet" href="./public/css/index.css">
+      <link rel="stylesheet" href="./components/navigation.css">
       <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
-      <link rel="stylesheet" href="./public/css/task_two.css">
 </head>
-
-<button type="button" class="btn btn-primary1" onclick="Time()" id="Time">Time</button>
-
-<body-one>
-
-      
-      <div class="btn-group" role="group" aria-label="Basic example">
-            <button type="button" class="btn btn-primary-two" id="Request" onclick="Request()">Request</button>
-
-            <button type="button" class="btn btn-primary-tree" onclick="Button()" id="Button">Button</button>
-
-            <button type="button" class="btn btn-primary-four" onclick="Error()" id="error">Sing in</button>
-
-          </div>
-
-</body-one>
-      <div class="section-task_one">
-            <button class="btn btn-primary button-task_one">Click</button>
-            <div class="square-section">
-
-            </div>
-      </div>
-
-<div class="section-task_two">
-      <button type="button" class="btn btn-warning button-task_two"  onclick="Alert()" id="Alert">Alert</button>
-
-</div>
+<body>
 
 <div class="section-task_five">
       <a href="./pages/list.html">List</a>
+
 </div>
 
-<script src="./public/js/Index.js"></script>
-<script src="./public/js/task_one.js"></script>
+</body>
+
+<script src="./public/js/index.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
```

Comment:
Move line 22 and 21 inside the </body> in line 18 and 17.
And line 17 with </div> move it to line 16.
Move </html> to line 20, and remove rest of empty lines from bottom of file.